### ISSUE TITLE
Use Issue/Pull-Request "Assignee" rather than comment.

### ISSUE THREAD

**Issue by stevenharman** [iamvery/lita-reviewme#25]
GitHub Issues and Pull Requests can be assigned to a user. What if rather than commenting, which is a pretty passive and indirect way of tagging someone to review a PR, we assigned the user to the PR.

We could leverage Octokit's `#update_issue(repo, issue_id, assignee: 'stevenharman')` API to get this done.

**Comment by iamvery** [comment]
I like the idea. My only concern is that feels more like they have some responsibility to merge the PR. On our current project, it's pretty common to have a PR assigned to someone on the project for final merge, but we still occasionally use _review me_ to solicit outside feedback on the code.

What do you think about that case?

**Comment by stevenharman** [comment]
Yeah, I can see that.

This mostly came from a desire for more explicitness, and also consistentcy with how other projects work - Rust, for example. I tend to prefer "you need to do the merge (b/c you likely have the most context to resolve any issues), but only after a :thumbsup: from someone else.

**Comment by iamvery** [comment]
Hmm, I like the sound of that. I wonder how it'd behave if the PR is already assigned to someone else? Maybe that's not a legit case. Meaning, if the PR is already assigned maybe it doesn't make sense to request such a review anyway... not entirely sure.

What do you think @kyfast? (sorry to loop you in without context...)

**Comment by stevenharman** [comment]
If it helps, you can assign multiple people to an issue/PR.

**Comment by iamvery** [comment]
Did not know that! Does the UI not support it? Having trouble figuring out how to do it.

**Comment by KyFaSt** [comment]
I like the idea of assigning the PR to a user. I wonder about it being confusing as to who should merge a pull-request but maybe it makes sense for the merger to be someone on the project who merges after the assigned reviewer gives their :+1:

**Comment by stevenharman** [comment]
@kyfast, that's my preference. In practice, it almost always seems the best person to merge is the author, or at least someone really close to the code on question

**Comment by iamvery** [comment]
Any concern with the bot assignment overriding an existing assignment?

**Comment by stevenharman** [comment]
Bot could fetch current Issue and just add someone to the list of assignees. ???

**Comment by iamvery** [comment]
I'm not convinced multiple assignments is possible for GH issues/prs. Are you sure about this?

**Comment by stevenharman** [comment]
You're right - single assignee only. I rechecked the API docs. I'm thinking that I had seen "list assignees" API and was thinking that was for an Issue. Mea Culpa.

**Comment by iamvery** [comment]
Bummer, cause this would be a decent use case for such a thing
